src/SFA.DAS.Commitments.Api.Client/SFA.DAS.Commitments.Api.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>SFA.DAS.Commitments.Api.Client</id>
-    <version>1.71.0</version>
```

Comment:
I don't think the nuspec is used used anymore when generating the package (it definitely ignores the supplied version number and auto generates a new one), so we don't need to update it.